Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
+ARG VERSION=1.0.3
```

Comment:
We could [ask github what the latest release is](https://github.com/crate-ci/gh-install/blob/master/v1/install.sh#L119)

Reply:
Or better, add a [pre-release replacement](https://github.com/crate-ci/typos/blob/master/Cargo.toml#L32) to auto-update this on release.

Reply:
I think we actually would want the container to be pinned with the current release for the repo - the reason is that a user can request an action for a specific release version, e.g.:

```yaml
uses: crate-ci/typos@1.0.3
```
If we have the container always use the latest, the release versions for the action won't have any meaning.

Reply:
Hmm the pre-release replacement is a cool idea... but it would mean only releases will work for the GitHub action (and not branches). Using branches isn't best practice but it's fairly common!

Reply:
Im not sure why.  Pre-release replacements just automatically update the `Dockerfile` in master when making a new release.  Branches can point to whatever version the branch author feels is appropriate.

Reply:
Could you show me how you would implement this? From what I can see, the pre-release replacement is changing a string to a version. This means that (unless the string already is a version, which defeats the purpose in the first place) it would mean that the Dockerfile in master won't build. I'll push changes soon and you can make a suggestion on the file to show me what you would like (I don't think I follow completely).

Reply:
In `Cargo.toml`. add the line
```toml
  {file="docker/Dockerfile", search="ARG VERSION=.*", replace="ARG VERSION={{version}}", min=1},
```
So when I release 1.0.4, `cargo release` will automatically update the Dockerfile` to 1.0.4.

Reply:
ohh I see! I totally misread the command to think that the `{{ version }}` needed to be in the Dockerfile directly (like a jinja2 template) but I see that it's referencing the actual version that is there. Yes this is a great idea, will update the PR to have this.

Reply:
All set.